### PR TITLE
adds valueRowCount property

### PR DESCRIPTION
**Pull request by lufego**
as discussed in https://github.com/andreypopp/react-textarea-autosize/pull/205

### CODE REVIEW

File: src/calculateNodeHeight.js [hunk 1]
```diff
@@ -96,6 +96,10 @@ export default function calculateNodeHeight(
   hiddenTextarea.value = 'x';
   const singleRowHeight = hiddenTextarea.scrollHeight - paddingSize;
 
+  // Stores the value's rows count rendered in `hiddenTextarea`,
+  // regardless if `maxRows` or `minRows` props are passed
+  const valueRowCount = Math.floor(height / singleRowHeight);
```
Comment (lufego) [inline src/calculateNodeHeight.js]: not sure about the naming here. Wether `fullRowCount` would make more sense

**Comment by Andarist** [comment]
Would this satisfy your use case? The thing I'm worried about that you probably will have to depend on ref to read this info, because it's not tightly correlated to `height` so it might change without `onHeightChange` firing.

**Comment by lufego** [comment]
Yes. This fixes our problem

> The thing I'm worried about that you probably will have to depend on ref to read this info, because it's not tightly correlated to height so it might change without onHeightChange firing.

Not sure if I understand. How would it change without me changing the value itself, and consequently triggering `onHeightChange `? Plus, I don't think it would be a problem since I only update the state of the parent component when `onHeightChange` is fired.

````js
handleHeightChange = (_, innerComponent) => {
    this.setState({
      textareaRowCount: innerComponent.rowCount,
      contentRowCount: innerComponent.valueRowCount,
    });
  };
````

**Comment by Andarist** [comment]
It can change if u i.e. have reached `maxRows` or `minRows` but type more text (or delete lines in second case) - then you won't be notified about new `valueRowCount` because `height` won't change.

**Comment by lufego** [comment]
but it should be fine at this point since I just need to know if the content is smaller or bigger than 1 row

**Comment by Andarist** [comment]
I've decided also to `expose` this in `this.props.onChange` - https://github.com/andreypopp/react-textarea-autosize/commit/582b796b19941f819b9142541f0ebc984243e8f5

I need to figure out some bundlers inconsistencies before releasing this though, hopefully should take no more than few days.

**Comment by lufego** [comment]
thanks @Andarist. We are looking forward for the release

**Comment by Andarist** [comment]
I've released new major version, please check it out.

**Pull request merged**